src/library.js

```diff
@@ -194,7 +194,7 @@ mergeInto(LibraryManager.library, {
   emscripten_resize_heap: 'ip',
   emscripten_resize_heap: function(requestedSize) {
     var oldSize = HEAPU8.length;
-    requestedSize = requestedSize >>> 0;
+    {{{ convertPtrToIdx('requestedSize') }}};
```

Comment:
Having a separate `convertPtrToIdx('variable')` function has the benefit that it avoids a redundant assignment to self in wasm2gb builds. I.e. if one writes

```js
variable = {{{ ptrToIdx('variable') }}};
```
Then it will generate `variable = variable;` in <2GB builds, and one will need to use Closure to fix that up.